src/screens/LoginCodeForm/index.tsx

```diff
@@ -114,6 +147,20 @@ const LoginCodeForm = ({ navigation }: LoginCodeFormProps) => {
             disabled={isLoading} customStyle={styles.input} />
           <NiInput caption={'Prénom'} value={firstname} onChangeText={setFirstname} type={'text'} required
             disabled={isLoading} customStyle={styles.input} />
+          <View style={styles.sectionContainer}>
+            <View style={styles.titleContainer}>
+              <Text style={styles.title}>Structure</Text>
+              <Text style={styles.required}>*</Text>
+            </View>
+            {!company.name && <NiSecondaryButton caption="Renseigner ma structure" onPress={chooseCompany}
+              font={FIRA_SANS_MEDIUM.MD} color={GREY[900]} />}
+            {company.name && <View style={styles.section}>
```

Comment:
tu peux rajouter un disabled is isLoading

Reply:
le texte ne se grise pas sur l'input de la structure durant l'appel

Reply:
corrigé 
J'ai simplifié la logique en utilisant un seul bouton